openpype/plugins/publish/integrate_thumbnail_ayon.py

```diff
@@ -5,7 +5,19 @@
     pull into a scene.
 
     This one is used only as image describing content of published item and
-    shows up only in Loader in right column section.
+    shows up only in Loader or Server UI in right column section.
+
+    Possible sources of thumbnail paths (OR):
+    - context.data.get("thumbnailPath")
+    (instance must have "published_representations")
+    - instance.data.get("thumbnailSource")
+    - instance.data.get("thumbnailPath")
+    - representation with 'thumbnail' name from "published_representations"
```

Comment:
thumbnailSource should not be used https://github.com/ynput/OpenPype/pull/6063
```suggestion
    Instance must have "published_representations" to
        be able to integrate thumbnail.
    Possible sources of thumbnail paths:
    - instance.data["thumbnailPath"]
    - representation with 'thumbnail' name from "published_representations"
    - context.data["thumbnailPath"]
    
```